bench.py

```diff
@@ -0,0 +1,84 @@
+import click
+import subprocess
+import os
+import sys
+import time
+import statistics
+
+
+@click.group()
+def cli():
+    """A benchmark suite for Scrapy."""
+    pass
+
+
+@cli.command()
+@click.option(
+    '--long',
```

Comment:
I think some other name would be better - ``long`` is a built-in in python 2, so it's better not to shadow it. Maybe you can call it ``--n-runs`` or something like this?